internal/series/series.go

```diff
@@ -52,6 +52,7 @@ var Series = map[string]SeriesInfo{
 	"yakkety": {true, Ubuntu, true, 1.103},
 	"zesty":   {true, Ubuntu, true, 1.104},
 	"artful":  {true, Ubuntu, true, 1.105},
+	"bionic":  {true, Ubuntu, true, 0},
```

Comment:
I'd suggest you should maybe put the boost number in a comment (I'd suggest 1.15)

Reply:
done